### PR TITLE
update 05-memory.md

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR introduces several changes to the 05-memory.md episode, including: 

1) Revising objectives, key points and questions to limit to a handful of items and reflect the revised lesson content. 

2) Moving the test of working memory to after a description of memory types. 

3) Removing extraneous examples of working memory limits (keeping some). 

4) Removing description of the serial position effect, as it introduces an added layer of complexity to the idea of working memory and cognitive load more generally. 

5) Reiterating the importance of formative assessment for helping learners make connections among concepts and reduce effective load.

6) Shortening description of different ways to use concept maps (while retaining the main points).

### CODE REVIEW

File: _episodes/05-memory.md [hunk 1]
```diff
@@ -1,123 +1,87 @@
...
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Explain the basis and implications of cognitive load theory and critique it."
+- "Identify when cognitive load is likely to limit learning."
 - "Use a concept map to plan a formative assessment strategy."
-- "Construct faded examples for use as classroom exercises and explain the problem-solving strategies those examples illustrate."
 keypoints:
-- "Most adults can store 7±2 items in short-term memory for a few seconds before loss."  
-- "Things seen together repeatedly are remembered (or mis-remembered) in chunks."  
+- "Most adults can store only a few items in short-term memory for a few seconds before loss."
```
Comment (lexnederbragt) [inline _episodes/05-memory.md]: before loss ->before they lose them again?

Comment (lexnederbragt) [inline _episodes/05-memory.md]: before loss ->before they lose them again?

File: _episodes/05-memory.md [hunk 2]
```diff
@@ -301,14 +242,6 @@ It posits that people have to deal with three things when they're learning:
...
-Unsurprisingly,
-it too has [been criticized][cognitive-load-crit],
-most particularly for being unfalsifiable.
-Critics of cognitive load theory say that
-since there's no way to tell in advance of an experiment whether something is germane or not,
-any result can be justified after the fact
-by labelling things that hurt performance as "extraneous"
-and things that don't as "germane".
 
 However,
```
Comment (lexnederbragt) [inline _episodes/05-memory.md]: With the removal of the critique part, 'however' is no longer needed. Maybe "In the framework of cognitive load, some predictions can be made!"?

File: _episodes/05-memory.md [hunk 3]
```diff
@@ -1,123 +1,87 @@
...
-[7±2 items][wikipedia-7]
-for a few seconds
-before things started to drop out.
+but it is slow to access.
+
+Our second layer of memory is called *short-term* or *working memory*. This is the 
+type of memory you use to actively think about things. It is much faster, but also much
+smaller: in 1956, Miller estimated that the average adult's working memory could hold
+[7±2 items][wikipedia-7] for a few seconds before things started to drop out.
 This is [why phone numbers are typically 7 or 8 digits long](https://www.quora.com/Why-did-Bell-Labs-create-phone-numbers-of-7-digits-10-digits):
```
Comment (ChristinaLK) [inline _episodes/05-memory.md]: This should be a reference link -- which is my bad b/c I'm pretty sure I added this one.

File: _episodes/05-memory.md (hunk 2 above)
Comment (ErinBecker) [inline _episodes/05-memory.md]: Thanks for catching this @lexnederbragt

**Comment by ErinBecker** [comment]
Merging based on review from @lexnederbragt and @ChristinaLK

**Pull request merged**